db_connection.py

```diff
@@ -27,25 +32,25 @@ def connect_db(self, connection_parameters):
                     port=connection_parameters['port']
                 )
                 self.cursor = self.conn.cursor()
-                print("Connection Success!")
+                logger.info("Connection Success!")
                 return 
             except OperationalError as e:
                 attempts += 1
-                print(f"Connection error: {e}. Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}.")
+                logger.error(f"Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}. Connection error: {e}")
```

Comment:
Can be shortened by using exc_info=True parameter.

Reply:
Resolved. Add exc_info parameter